lk2nd/device/menu/menu.c

```diff
@@ -228,6 +242,11 @@ void display_fastboot_menu(void)
 	if (lk2nd_dev.bootloader)
 		fbcon_printf_ln(SILVER, y, incr, false, " Bootloader:  %s", lk2nd_dev.bootloader);
 #endif
+#if SLOTS_SUPPORTED
+		int current_active_slot = partition_find_active_slot();
+		fbcon_printfn(SILVER, y, incr, false, " Active Slot:  ");
+		fbcon_printf_ln(current_active_slot ? GREEN : RED, y, incr, false, "              %s", current_active_slot ? "B" : "A");
+#endif
```

Comment:
It's unclear to me why A/B should be color-coded, nor why B should be green/"good" and A should be red/"bad"

Reply:
It just looks cool, lol.

No color is good/bad for me it just looks cool as said but we can keep it silver if you like it more